portality/static/js/edges/publisher.update_requests.edge.js

```diff
@@ -278,17 +280,46 @@ $.extend(true, doaj, {
                 template: edges.bs3.newFacetview(),
                 search_url: search_url,
                 manageUrl: true,
-                openingQuery : es.newQuery({
-                    sort: {"field" : "last_manual_update", "order" : "asc"}
+                openingQuery: es.newQuery({
+                    sort: {"field": "last_manual_update", "order": "asc"}
                 }),
                 components: components,
-                callbacks : {
-                    "edges:query-fail" : function() {
+                callbacks: {
+                    "edges:query-fail": function () {
                         alert("There was an unexpected error.  Please reload the page and try again.  If the issue persists please contact us.");
                     }
                 }
             });
             doaj.publisherUpdatesSearch.activeEdges[selector] = e;
+
+            $(selector).on("edges:post-render", function () {
+                $(".delete_suggestion_link").unbind("click").click(function (event) {
+                    console.log("click registered")
+                    event.preventDefault();
+
+                    function success_callback(data) {
+                        alert("The update request was successfully deleted");
+                        doaj.publisherUpdatesSearch.activeEdges[selector].cycle();
+                    }
+
+                    function error_callback() {
+                        alert("There was an error deleting the update request")
+                    }
+
+                    var c = confirm("Are you really really sure?  You can't undo this operation!");
+                    if (c) {
+                        var href = $(this).attr("href");
+                        var obj = {"delete": "true"};
```

Comment:
The original function, which I somehow managed to miss during the Edges upgrade, didn't have this `obj` bit, is it necessary?